python/pyspark/sql/streaming/stateful_processor.py

```diff
@@ -50,19 +50,11 @@ def exists(self) -> bool:
         """
         return self._value_state_client.exists(self._state_name)
 
-    def get(self) -> Optional[Row]:
+    def get(self) -> Optional[Tuple]:
         """
         Get the state value if it exists. Returns None if the state variable does not have a value.
         """
-        value = self._value_state_client.get(self._state_name)
-        if value is None:
-            return None
-        schema = self.schema
-        if isinstance(schema, str):
-            schema = cast(StructType, _parse_datatype_string(schema))
-        # Create the Row using the values and schema fields
-        row = _create_row(schema.fieldNames(), value)
-        return row
+        return self._value_state_client.get(self._state_name)
 
     def update(self, new_value: Any) -> None:
```

Comment:
nit: mind updating this to Tuple as well? Please address this as follow-up PR as this PR has been around for a long time and it's just a tiny change.

Reply:
Sure, will address this in the next PR, thanks!